enso.cfg

```diff
@@ -0,0 +1,14 @@
+[#]
+sets = ["enso_diags"]
+case_id = "Regression on Nino3.4"
```

Comment:
Is this the correct `case_id`? It seems like usually it is something like `GPCP_v2.2`.